.check-author.yml

```diff
@@ -23,3 +23,5 @@ mapping:
   "Ingolf Steinhardt <info@e-spin.de>":
     - "xantippe <info@e-spin.de>"
     - "zonky2 <info@e-spin.de>"
+  "Cliff Parnitzky <contao@cliff-parnitzky.de>":
+    - "cliffparnitzky <contao@cliff-parnitzky.de>"
```

Comment:
You are push as "Cliff Parnitzky <github@cliff-parnitzky.de>".

Reply:
You should check your local github settings

Reply:
okay, i will change this